CHANGELOG.md

```diff
@@ -22,7 +22,8 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
-* Use the SWIFT_VERSION when linting pods.  
+* Use the SWIFT_VERSION when linting pods. To lint with Swift 3.0
+  add a Swift Version file. `echo "3.0" >> .swift_version`.   
```

Comment:
I think this should say `.swift-version` not `.swift_version`, at least based on my empirical evidence.

Reply:
Doesn't seem to work for Swift 4.2